### DISCUSSION TITLE
imgui_impl_opengl3_loader.h definition errors

### DISCUSSION THREAD

**Discussion by Fabxx** [ocornut/imgui#4977] [New Users Build/Link/Run issues ONLY!]
compile log:
[log.txt](https://github.com/ocornut/imgui/files/7979454/log.txt)

any suggestions?

**Comment by ocornut** [comment]
You are omitting too much details to help.
To clarify you SHOULD NEVER USE the loader.h file directly. This is only used by our loader.
